packages/foundations/scripts/generate-icon-fonts/index.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-await-in-loop */
+
+const FSE = require('fs-extra');
+const { startProgram } = require('../program.js');
+const { cleanIcons } = require('../clean-icons/index.js');
+const { gatherIcons } = require('./gather-icons.js');
+const { svgToFont } = require('./svg-to-font.js');
+
+const options = [
+	{
+		name: 'ignoreGlobs',
+		short: 'ig',
+		description: 'Path icon glob to exclude into the fonts',
```

Comment:
```suggestion
		description: 'Path icon glob to exclude from the fonts',
```

https://github.com/db-ui/mono/pull/1490/files#r1310078200